docker/oracle/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Can you remove this & all the other docker related pieces

Reply:
done